packages/core/src/vendor-types/git.vendor-types.ts

```diff
@@ -43,25 +44,60 @@ export const GitDataTypes = {
     },
     ...gitCommonTypeInfo,
   }),
-  BranchName: createDmnoDataType({
-    typeLabel: 'git/branchName',
-    typeDescription: 'Name of a git branch',
-    externalDocs: {
-      description: 'Git branches (Atlassian docs)',
-      url: 'https://www.atlassian.com/git/tutorials/using-branches',
-    },
-    ...gitCommonTypeInfo,
-  }),
-  CommitSha: createDmnoDataType({
-    typeLabel: 'git/commitSha',
-    typeDescription: 'Unique id of a specific git commit - also known as “SHA” or “hash”',
-    // full SHA is 40 chars, but it is common to use shortened versions, so we may want multiple types or options
-    externalDocs: {
-      description: 'What are Git Hashes? (Graphite docs)',
-      url: 'https://graphite.dev/guides/git-hashing',
-    },
-    ...gitCommonTypeInfo,
-  }),
+  BranchName: createDmnoDataType(
+    (opts?: {
```

Comment:
This is what prompted the changes. I wanted to be able to toggle having a built-in `value` (resolver) based on the presence of an instance option.